source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,284 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+In MongoDB, you can use a text search to retrieve documents that
+contain a term or a phrase in a specified field. A term is a sequence of
+characters that excludes whitespace characters. A phrase is a sequence
+of terms that with any number of whitespace characters.
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+In the following sections, you run text searches on the
+``fast_and_furious_movies`` collection. Each section uses ``collection``
+to refer to the ``MongoCollection`` instance of the
+``fast_and_furious_movies`` collection. 
+
+This collection contains the following documents that represent one of
+the several movies that are part of the Fast and Furious movie franchise
+and tags they are mostly associated with:
+
+.. code-block:: json
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] } 
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Text Index
+----------
+
+You must create a **text index** before running a text search. A text
+index specifies the string or string array field for the query to run a
+text search on. A collection can only contain one text index.
```

Comment:
I don't think adding the preposition "on" at the end is grammatically correct because it doesn't relate the noun/verb to the preposition. Instead, consider "field on which to run a text search."

Reply:
In the current version, I see:
"A text index specifies the string or string array field for the query to run a text search."

I don't think "to" is the right preposition. If the sentence was "To run a text search, do X", then "to" is appropriate. In this case, you should use "on which to run" as recommended in my prior comment because it refers to the field.